CombatTagPlus/src/main/resources/config.yml

```diff
@@ -1,15 +1,25 @@
 # Don't touch this. It's here to determine whether you need to refresh your config.
-config-version: 20
+config-version: 21
 
 # The duration in seconds that both the attacker and victim should be tagged in combat.
 tag-duration: 15
 
-# This message is displayed to both the attacker and victim when newly tagged. Set this to '' to display nothing. {opponent} is the other player.
-tag-message: '&cYou have engaged in combat with &b{opponent}&c. Type &b/ct &cto check your timer.'
```

Comment:
This way you will break all existing servers config. Plugins should always be backwards compatible config wise if possible, so you'd want to keep this option and only use the new messages if they actually exist, if not fallback to the old behavior